cmd/ocfclient/main.go

```diff
@@ -553,27 +653,9 @@ func printMenu() {
 var (
 	CertIdentity = "00000000-0000-0000-0000-000000000001"
 
-	MfgCert = []byte(`-----BEGIN CERTIFICATE-----
-MIIB9zCCAZygAwIBAgIRAOwIWPAt19w7DswoszkVIEIwCgYIKoZIzj0EAwIwEzER
-MA8GA1UEChMIVGVzdCBPUkcwHhcNMTkwNTAyMjAwNjQ4WhcNMjkwMzEwMjAwNjQ4
-WjBHMREwDwYDVQQKEwhUZXN0IE9SRzEyMDAGA1UEAxMpdXVpZDpiNWEyYTQyZS1i
-Mjg1LTQyZjEtYTM2Yi0wMzRjOGZjOGVmZDUwWTATBgcqhkjOPQIBBggqhkjOPQMB
-BwNCAAQS4eiM0HNPROaiAknAOW08mpCKDQmpMUkywdcNKoJv1qnEedBhWne7Z0jq
-zSYQbyqyIVGujnI3K7C63NRbQOXQo4GcMIGZMA4GA1UdDwEB/wQEAwIDiDAzBgNV
-HSUELDAqBggrBgEFBQcDAQYIKwYBBQUHAwIGCCsGAQUFBwMBBgorBgEEAYLefAEG
-MAwGA1UdEwEB/wQCMAAwRAYDVR0RBD0wO4IJbG9jYWxob3N0hwQAAAAAhwR/AAAB
-hxAAAAAAAAAAAAAAAAAAAAAAhxAAAAAAAAAAAAAAAAAAAAABMAoGCCqGSM49BAMC
-A0kAMEYCIQDuhl6zj6gl2YZbBzh7Th0uu5izdISuU/ESG+vHrEp7xwIhANCA7tSt
-aBlce+W76mTIhwMFXQfyF3awWIGjOcfTV8pU
------END CERTIFICATE-----
-`)
+	MfgCert = []byte{}
 
-	MfgKey = []byte(`-----BEGIN EC PRIVATE KEY-----
-MHcCAQEEIMPeADszZajrkEy4YvACwcbR0pSdlKG+m8ALJ6lj/ykdoAoGCCqGSM49
-AwEHoUQDQgAEEuHojNBzT0TmogJJwDltPJqQig0JqTFJMsHXDSqCb9apxHnQYVp3
-u2dI6s0mEG8qsiFRro5yNyuwutzUW0Dl0A==
------END EC PRIVATE KEY-----
-`)
+	MfgKey = []byte{}
 
 	MfgTrustedCA = []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
@Trishia All this certs must be generated if they are not provided - similar as for identity certs. If someone use this sample app in production this can cause very big security issue.

Reply:
Execution of this tool shall generate CA and persist them on disk from where they are loaded when started for the 2nd time.

Reply:
@ondrejtomcik @jkralik 
OK, I will change your comments 2days later. I'm busy now until tomorrow.